Lib/asyncio/runners.py

```diff
@@ -84,9 +84,13 @@ def get_loop(self):
         return self._loop
 
     def run(self, coro, *, context=None):
-        """Run a coroutine inside the embedded event loop."""
-        if not coroutines.iscoroutine(coro):
-            raise ValueError("a coroutine was expected, got {!r}".format(coro))
+        """Run code in the embedded event loop.
+
+        The argument can be an awaitable, coroutine, Future, or Task.
+        If the argument is a coroutine, it is wrapped in a Task.
+
+        Return the coro's result or raise an exception.
+        """
```

Comment:
```suggestion
        The argument can be any awaitable object.
        
        If the argument is a coroutine, it is wrapped in a Task.

        Return the awaitable's result or raise an exception.
        """
```

I know that the parameter is called "coro" but I'd prefer not using "coro's" in the docstring.